plugin-api/pom.xml

```diff
@@ -91,6 +97,27 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
```

Comment:
We can use a class to get data from System env. For example:
```java

public class EnvInfo {

	private Map<String,String> map;

	ignore getter setter


	public String getInfo(String key){
		return this.map.get(key);
	}

}

```
This class can be isolated from getting environment variables.

In unit test:

```java

EnvInfo envInfo = new EnvInfo();

// mock data in system environment
envInfo.setMap();

```

Other classes can get data from EnvInfo.

Reply:
good idea, i'll change it